workflow/overview.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: "Workflow Builder Overview"
+sidebarTitle: "Overview"
+---
+
+Our browser-based **Workflow Builder (Editor)** makes it simple to create personalized, automated messaging workflows—no coding required.
```

Comment:
Let's choose a single name here either editor or builder. Also no need to specify it's browser based

Reply:
also, workflow editor is used to modify code based workflows. So this statement is in accurate.

Reply:
Addressed.